build/stf-run-ci/tasks/setup_stf_from_bundles.yml

```diff
@@ -88,10 +84,30 @@
     fail_msg: "Bundle path(s) not set. __smart_gateway_bundle_image_path is '{{ __smart_gateway_bundle_image_path }}' and __service_telemetry_bundle_image_path is '{{ __service_telemetry_bundle_image_path }}'. Both values need to be set."
     success_msg: "Bundle paths are defined and not None"
 
+# TODO: Fix this check
+- name: "Is pull_secret defined?"
+  ansible.builtin.debug:
+    msg: "{{ pull_secret is defined }}"
+
+- name: "What is the value of pull_secret?"
+  ansible.builtin.debug:
+    var: pull_secret
```

Comment:
Remove this before merging, we shouldn't output the pull secret

Reply:
Basically, none of this is needed, and will be removed.